core/src/main/scala/tofu/Timeout.scala

```diff
@@ -13,7 +14,7 @@ trait Timeout[F[_]] {
 }
 
 object Timeout {
-  def concurrent[F[_]](implicit F: Concurrent[F], timer: Timer[F]): Timeout[F] =
+  implicit def concurrent[F[_]](implicit F: Concurrent[F], timer: Timer[F], nonTofu: NonTofu[F]): Timeout[F] =
     new Timeout[F] {
       override def timeoutTo[A](fa: F[A], after: FiniteDuration, fallback: F[A]): F[A] =
         F.race(fa, timer.sleep(after) *> fallback).map(_.merge)
```

Comment:
Isn't related to this change, but shouldn't it be a race between `fa` and `sleep`?
Something like: `F.race(timer.sleep(after), fa).getOrElseF(fallback)`?

Reply:
Yeah definitely. @FunFunFine can you include that fix as well?

Reply:
Will lock it with "request changes for now"